<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Bayesian Network for Synthetic Data â€¢ cvdRiskData</title>
<!-- jquery --><script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha384-nrOSfDHtoPMzJHjVTdCopGqIqeYETSXhZDFyniQ8ZHcVy08QesyHcnOUpMpqnmWq" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script><!-- Font Awesome icons --><link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">
<!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../jquery.sticky-kit.min.js"></script><script src="../pkgdown.js"></script><!-- mathjax --><script src="https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-vignette">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="../index.html">cvdRiskData</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="..//index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/build_bayesian_network.html">Bayesian Network for Synthetic Data</a>
    </li>
    <li>
      <a href="../articles/generate_data_from_network.html">Building Data from our Bayesian Network</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><div class="row">
  <div class="col-md-9">
    <div class="page-header toc-ignore">
      <h1>Bayesian Network for Synthetic Data</h1>
                        <h4 class="author">Ted Laderas</h4>
            
            <h4 class="date">2018-03-20</h4>
          </div>

    
    
<div class="contents">
<p>In this file, I start generating the Bayesian Network from our data using the <code>gRain</code> package.</p>
<div id="basic-synthetic-dataset" class="section level2">
<h2 class="hasAnchor">
<a href="#basic-synthetic-dataset" class="anchor"></a>Basic Synthetic Dataset</h2>
<p>The first variable we add is <code>age</code>. Then I add <code>smoking</code>, which is associated with hypertension.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">yn &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"N"</span>,<span class="st">"Y"</span>)

<span class="co">#this is the true conditional prob table for HTNtreat given HTN status</span>
<span class="co">#condHTNtreat &lt;- t(HTNtreat/rowSums(HTNtreat))</span>

<span class="co">#I adjusted according to David's suggestions</span>
condHTNtreat &lt;-<span class="st"> </span><span class="kw">t</span>(<span class="kw">matrix</span>(<span class="dt">nrow=</span><span class="dv">2</span>,<span class="kw">c</span>(<span class="dv">100</span>,<span class="dv">0</span>,<span class="dv">40</span>,<span class="dv">60</span>), <span class="dt">byrow =</span> <span class="ot">TRUE</span>, <span class="dt">dimnames=</span><span class="kw">list</span>(yn,yn)))

## Age probabilities are calculated from our cohort
ageLevels &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"0-20"</span>,<span class="st">"20-40"</span>, <span class="st">"40-55"</span>, <span class="st">"55-70"</span>,<span class="st">"70-90"</span>)

ageProbMat &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="kw">system.file</span>(<span class="st">"cpt"</span>, <span class="st">"age_probabilities.csv"</span>, <span class="dt">package=</span><span class="st">"cvdRiskData"</span>))
ageProb &lt;-<span class="st"> </span>ageProbMat<span class="op">$</span>Freq
<span class="kw">names</span>(ageProb) &lt;-<span class="st"> </span>ageProbMat<span class="op">$</span>out

ageProb</code></pre></div>
<pre><code>##     (0,20]    (20,40]    (40,55]    (55,70]    (70,90] 
## 0.08744536 0.29710261 0.25755875 0.23272094 0.12517234</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">a &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>age, <span class="dt">values=</span>ageProb<span class="op">*</span><span class="dv">100</span>, <span class="dt">levels=</span>ageLevels)

## These conditional probabilities are derived from our cohort
condHTNAgeprob &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="kw">system.file</span>(<span class="st">"cpt"</span>, <span class="st">"age_htn_cpt.csv"</span>, <span class="dt">package=</span><span class="st">"cvdRiskData"</span>), <span class="dt">check.names =</span> <span class="ot">FALSE</span>)

## Here are the conditional age|htn probabilities
condHTNAgeprob</code></pre></div>
<pre><code>##       (0,20]   (20,40]   (40,55]  (55,70]   (70,90]
## 1 0.97665607 0.8516853 0.7206523 0.523628 0.3901981
## 2 0.02334393 0.1483147 0.2793477 0.476372 0.6098019</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">rownames</span>(condHTNAgeprob) &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">1</span>)
<span class="kw">colnames</span>(condHTNAgeprob) &lt;-<span class="st"> </span>ageLevels

## Compile our CPTs so far
a.htn &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>htn<span class="op">+</span>age, <span class="dt">values=</span><span class="kw">as.matrix</span>(condHTNAgeprob), <span class="dt">levels=</span>yn)
htn.treat &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>treat<span class="op">+</span>htn, <span class="dt">values=</span>condHTNtreat, <span class="dt">levels=</span>yn)

##Smoking|age CPT - peaks at 25% smoking for 55-70 category
a.s &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>smoking<span class="op">+</span>age, <span class="dt">values=</span><span class="kw">c</span>(<span class="dv">95</span>,<span class="dv">5</span>,<span class="dv">90</span>,<span class="dv">10</span>,<span class="dv">85</span>,<span class="dv">15</span>,<span class="dv">75</span>,<span class="dv">25</span>,<span class="dv">90</span>,<span class="dv">10</span>),<span class="dt">levels=</span>yn)
##Smoking htn, if you smoke, you have 67% prob of having hypertension
s.h &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>htn<span class="op">+</span>smoking, <span class="dt">values=</span><span class="kw">c</span>(<span class="dv">50</span>,<span class="dv">50</span>,<span class="dv">33</span>,<span class="dv">67</span>))</code></pre></div>
<p>Now we load in probabilities for race, which was calculated from our patient cohort.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">pRaceMat &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="kw">system.file</span>(<span class="st">"cpt"</span>, <span class="st">"race_probabilities.csv"</span>, <span class="dt">package=</span><span class="st">"cvdRiskData"</span>))

pRace &lt;-<span class="st"> </span>pRaceMat<span class="op">$</span>Freq
<span class="kw">names</span>(pRace) &lt;-<span class="st"> </span>pRaceMat<span class="op">$</span>Var1

pRace</code></pre></div>
<pre><code>##       AmInd    Asian/PI  Black/AfAm       White 
## 0.005527018 0.191968947 0.053266109 0.749237925</code></pre>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">race &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>race, <span class="dt">values=</span>pRace, <span class="dt">levels=</span><span class="kw">names</span>(pRace))

<span class="co">#These probabilities are estimated for US patients overall (from CDC report)</span>
bmiLevels &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"15-18"</span>,<span class="st">"18-25"</span>, <span class="st">"25-31"</span>,<span class="st">"31+"</span>)
bmiProb &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="fl">0.16</span>, <span class="fl">0.68</span>, <span class="fl">0.10</span>, <span class="fl">0.06</span>)

bmi &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>bmi, <span class="dt">values=</span>bmiProb, <span class="dt">levels=</span>bmiLevels)

<span class="co">#conditional CPT for BMI given race</span>
<span class="co">#these values were estimated from CDC report</span>
<span class="co">#If you are Black/AfAm or White, have a higher </span>
<span class="co">#probability of being in a higher BMI category</span>
valuesBMI &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="dv">97</span>, <span class="dv">3</span>, <span class="dv">98</span>, <span class="dv">1</span>, <span class="dv">98</span>, <span class="dv">2</span>, <span class="dv">99</span>, <span class="dv">1</span>,
            <span class="dv">88</span>, <span class="dv">12</span>, <span class="dv">92</span>,<span class="dv">8</span>, <span class="dv">92</span>, <span class="dv">8</span>, <span class="dv">96</span>, <span class="dv">4</span>,
            <span class="dv">55</span>, <span class="dv">45</span>, <span class="dv">70</span>, <span class="dv">30</span>, <span class="dv">70</span>, <span class="dv">30</span>, <span class="dv">85</span>, <span class="dv">15</span>,
            <span class="dv">20</span>, <span class="dv">80</span>, <span class="dv">33</span>, <span class="dv">66</span>, <span class="dv">33</span>, <span class="dv">66</span>, <span class="dv">65</span>, <span class="dv">35</span>)

##t2d.race.bmi is a joint cpt: p(t2d|race,bmi)
t2d.race.bmi &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>t2d<span class="op">|</span>race<span class="op">:</span>bmi, <span class="dt">values=</span><span class="kw">c</span>(valuesBMI), <span class="dt">levels=</span>yn)
<span class="co">#t2d.cvd &lt;- cptable(~cvd|t2d, values=c(95, 5, 5, 95), levels= yn)</span>

##incidence of smoking is 15%
smoking &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>smoking, <span class="dt">values=</span><span class="kw">c</span>(<span class="dv">85</span>, <span class="dv">15</span>), <span class="dt">levels=</span>yn)</code></pre></div>
</div>
<div id="selected-snps" class="section level2">
<h2 class="hasAnchor">
<a href="#selected-snps" class="anchor"></a>Selected SNPs</h2>
<p>These first 2 snps are on the same chromosome and so I will model them as having identical risk, and having both does not increase risk. I am limiting myself to the homozygous variants (excluding the heterozygous variants) in order to simplify things. I also only consider 6 genotypes overall.</p>
<p><a href="http://snpedia.com/index.php/Rs10757278">Rs10757278</a> (Chr 9, 22124478). (A;A) 0.78 x risk of heart disease, (A;G) 1.3 x risk, (G;G) 1.6x risk for heart disease. Co-morbidity: diabetes</p>
<p><a href="http://snpedia.com/index.php/Rs1333049">Rs1333049</a> (Chr 9, 22125504). (G;G) Normal (WT); (C;G) 1.5 increased risk; (C;C) 1.9x increased risk.</p>
<p>These last 2 snps are on different chromosomes for the first. Again, limiting to the homozygous cases to simplify things.</p>
<p><a href="http://snpedia.com/index.php/Rs4665058">Rs4665058</a>, 4x risk if have two copies of the SNP (A;A), 2 X (A;C) and (C;C) - wild type. Chr 2, 159333698</p>
<p><a href="https://www.snpedia.com/index.php/Rs8055236">rs8055236</a> (Chr 16, 83178793) (T;T) Normal (WT); (G;T) 1.9x risk, (G;G) 2.2x increased risk</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">raceNames &lt;-<span class="st"> </span><span class="kw">names</span>(pRace)

<span class="co">#Conversion table between aggregate genotype and SNP values</span>
snpLookup &lt;-<span class="st"> </span><span class="kw">list</span>(<span class="st">"0000"</span> =<span class="st"> </span><span class="kw">c</span>(<span class="st">"AA"</span>, <span class="st">"CC"</span>, <span class="st">"CC"</span>, <span class="st">"TT"</span>),
     <span class="st">"0001"</span> =<span class="st"> </span><span class="kw">c</span>(<span class="st">"AA"</span>, <span class="st">"CC"</span>, <span class="st">"CC"</span>, <span class="st">"GG"</span>),
     <span class="st">"0010"</span> =<span class="st"> </span><span class="kw">c</span>(<span class="st">"AA"</span>, <span class="st">"CC"</span>, <span class="st">"AA"</span>, <span class="st">"TT"</span>),
     <span class="st">"1100"</span> =<span class="st"> </span><span class="kw">c</span>(<span class="st">"GG"</span>, <span class="st">"GG"</span>, <span class="st">"CC"</span>, <span class="st">"TT"</span>),
     <span class="st">"1110"</span> =<span class="st"> </span><span class="kw">c</span>(<span class="st">"GG"</span>, <span class="st">"GG"</span>, <span class="st">"AA"</span>, <span class="st">"GG"</span>),
     <span class="st">"1111"</span> =<span class="st"> </span><span class="kw">c</span>(<span class="st">"GG"</span>, <span class="st">"GG"</span>, <span class="st">"AA"</span>, <span class="st">"GG"</span>))

snpNames &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"rs10757278"</span>, <span class="st">"rs1333049"</span>, <span class="st">"rs4665058"</span>, <span class="st">"rs8055236"</span>)

snpLookup &lt;-<span class="st"> </span><span class="kw">lapply</span>(snpLookup, <span class="cf">function</span>(x) {<span class="kw">names</span>(x) &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"rs10757278"</span>, <span class="st">"rs1333049"</span>, <span class="st">"rs4665058"</span>, <span class="st">"rs8055236"</span>); <span class="kw">return</span>(x)})

rs10757278race =<span class="st"> </span><span class="kw">matrix</span>(<span class="dt">nrow=</span><span class="dv">2</span>, <span class="dt">byrow =</span> <span class="ot">TRUE</span>, <span class="dt">data=</span><span class="kw">c</span>(<span class="dv">21</span>, <span class="dv">24</span>, <span class="dv">80</span>, <span class="dv">30</span>, 
                                                     <span class="dv">79</span>, <span class="dv">76</span>, <span class="dv">20</span>, <span class="dv">70</span>), 
                        <span class="dt">dimnames =</span> <span class="kw">list</span>( <span class="kw">c</span>(<span class="st">"AA"</span>,<span class="st">"GG"</span>),raceNames))<span class="op">/</span><span class="dv">100</span>

r1 &lt;-<span class="st"> </span>rs10757278race

<span class="co">#p("AA"|race) = c(10, 10, 10, 10) </span>
<span class="co">#p("CC"|race) = c(80,80,80,80)</span>

rs4665058race =<span class="st"> </span>
<span class="st">  </span><span class="kw">matrix</span>(<span class="dt">nrow=</span><span class="dv">2</span>, <span class="dt">data=</span> <span class="kw">c</span>(<span class="dv">10</span>, <span class="dv">10</span>, <span class="dv">10</span>, <span class="dv">10</span>, <span class="dv">90</span>,<span class="dv">90</span>,<span class="dv">90</span>,<span class="dv">90</span>), <span class="dt">byrow=</span><span class="ot">TRUE</span>, 
         <span class="dt">dimnames=</span> <span class="kw">list</span>(<span class="kw">c</span>(<span class="st">"AA"</span>,<span class="st">"CC"</span>), raceNames ))<span class="op">/</span><span class="dv">100</span>

r2 &lt;-<span class="st"> </span>rs4665058race

<span class="co">#p("GG"|race) = c(62, 65, 20, 62)</span>
<span class="co">#P("TT"|race) = c(38, 35, 80, 38)</span>

rs8055236race =<span class="st"> </span><span class="kw">matrix</span>(<span class="dt">nrow=</span><span class="dv">2</span>, <span class="dt">byrow=</span><span class="ot">TRUE</span>, <span class="dt">data=</span><span class="kw">c</span>(<span class="dv">62</span>, <span class="dv">65</span>, <span class="dv">20</span>, <span class="dv">40</span>, 
                                                  <span class="dv">38</span>, <span class="dv">35</span>, <span class="dv">80</span>, <span class="dv">60</span>),
                       <span class="dt">dimnames =</span> <span class="kw">list</span>(<span class="kw">c</span>(<span class="st">"GG"</span>,<span class="st">"TT"</span>), raceNames))<span class="op">/</span><span class="dv">100</span>
r3 &lt;-<span class="st"> </span>rs8055236race

genotypes &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"1111"</span>, <span class="st">"1110"</span>, <span class="st">"1100"</span>, <span class="st">"0010"</span>, <span class="st">"0001"</span>, <span class="st">"0000"</span>)

probList &lt;-<span class="st"> </span><span class="kw">list</span>()
probList[[<span class="st">"0000"</span>]] &lt;-<span class="st"> </span>r1[<span class="dv">2</span>,] <span class="op">*</span><span class="st"> </span>r2[<span class="dv">2</span>,] <span class="op">*</span><span class="st"> </span>r3[<span class="dv">2</span>,]
probList[[<span class="st">"1100"</span>]] &lt;-<span class="st"> </span>r1[<span class="dv">1</span>,] <span class="op">*</span><span class="st"> </span>r2[<span class="dv">2</span>,] <span class="op">*</span><span class="st"> </span>r3[<span class="dv">2</span>,]
probList[[<span class="st">"1110"</span>]] &lt;-<span class="st"> </span>r1[<span class="dv">1</span>,] <span class="op">*</span><span class="st"> </span>r2[<span class="dv">1</span>,] <span class="op">*</span><span class="st"> </span>r3[<span class="dv">2</span>,]
probList[[<span class="st">"0010"</span>]] &lt;-<span class="st"> </span>r1[<span class="dv">2</span>,] <span class="op">*</span><span class="st"> </span>r2[<span class="dv">1</span>,] <span class="op">*</span><span class="st"> </span>r3[<span class="dv">2</span>,]
probList[[<span class="st">"0001"</span>]] &lt;-<span class="st"> </span>r1[<span class="dv">2</span>,] <span class="op">*</span><span class="st"> </span>r2[<span class="dv">2</span>,] <span class="op">*</span><span class="st"> </span>r3[<span class="dv">1</span>,]
probList[[<span class="st">"1111"</span>]] &lt;-<span class="st"> </span>r1[<span class="dv">1</span>,] <span class="op">*</span><span class="st"> </span>r2[<span class="dv">1</span>,] <span class="op">*</span><span class="st"> </span>r3[<span class="dv">1</span>,]

cptgenoRace &lt;-<span class="st"> </span><span class="kw">do.call</span>(rbind,probList)
cptgenoRace &lt;-<span class="st"> </span>cptgenoRace[genotypes,]

race.genotype &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>genotype<span class="op">|</span>race, <span class="dt">values=</span>cptgenoRace, <span class="dt">levels=</span>genotypes, <span class="dt">normalize=</span><span class="ot">TRUE</span>)

##Gender is unbalanced
gender &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>gender, <span class="kw">c</span>(<span class="dv">45</span>,<span class="dv">55</span>), <span class="dt">levels=</span><span class="kw">c</span>(<span class="st">"M"</span>,<span class="st">"F"</span>))

##Two SNPs are associated with higher cholesterol
tcholStates &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">"&lt;160"</span>, <span class="st">"160-199"</span>, <span class="st">"200-239"</span>,<span class="st">"240+"</span>)
genotype.tchol &lt;-<span class="st"> </span><span class="kw">cptable</span>(<span class="op">~</span>tchol<span class="op">|</span>genotype, 
                          <span class="dt">values =</span> <span class="kw">c</span>(<span class="dv">5</span>,<span class="dv">10</span>,<span class="dv">30</span>,<span class="dv">55</span>,
                                     <span class="dv">10</span>,<span class="dv">20</span>,<span class="dv">30</span>,<span class="dv">40</span>,
                                     <span class="dv">15</span>,<span class="dv">20</span>,<span class="dv">45</span>,<span class="dv">30</span>,
                                     <span class="dv">15</span>,<span class="dv">20</span>,<span class="dv">45</span>,<span class="dv">30</span>,
                                     <span class="dv">30</span>,<span class="dv">60</span>,<span class="dv">5</span>,<span class="dv">5</span>,
                                     <span class="dv">30</span>,<span class="dv">60</span>,<span class="dv">5</span>,<span class="dv">5</span>), <span class="dt">levels=</span>tcholStates)</code></pre></div>
<p>Finally, we compile our Bayesian Network and save it.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">plist &lt;-<span class="st"> </span><span class="kw">compileCPT</span>(<span class="kw">list</span>(a, a.htn, htn.treat, smoking, s.h, race,
                         bmi, t2d.race.bmi, race.genotype, 
                         genotype.tchol,
                         gender))

cvd_bayes_net &lt;-<span class="st"> </span><span class="kw">compile</span>(<span class="kw">grain</span>(plist))
devtools<span class="op">::</span><span class="kw"><a href="http://www.rdocumentation.org/packages/devtools/topics/use_data">use_data</a></span>(cvd_bayes_net, <span class="dt">overwrite =</span> <span class="ot">TRUE</span>)</code></pre></div>
<pre><code>## Saving cvd_bayes_net as cvd_bayes_net.rda to /Users/roshi/Code/cvdRiskData/data</code></pre>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#basic-synthetic-dataset">Basic Synthetic Dataset</a></li>
      <li><a href="#selected-snps">Selected SNPs</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by Ted Laderas.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="http://hadley.github.io/pkgdown/">pkgdown</a>.</p>
</div>

      </footer>
</div>

  </body>
</html>
